tools/qlog/src/lib.rs

```diff
@@ -1194,8 +1195,8 @@ pub enum EventData {
     ServerListening {
         ip_v4: Option<String>,
         ip_v6: Option<String>,
-        port_v4: u64,
-        port_v6: u64,
+        port_v4: u32,
```

Comment:
Can we use `u16` for port numbers? (same for L1213/1214)

Reply:
these are taken straight from https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#name-server_listening

Reply:
ah ok. I will look qlog definitions. lgtm